scripts/mdtopdf.sh

```diff
@@ -95,21 +100,23 @@ function main
      sed -i -r 's,\n,\\n,g' "${TEMP_FILE}"
      sed -i -r 's,_,\_,g' "${TEMP_FILE}"
 
-     #Fix for all local links to not mislead Pandoc into grabbing other files when there is nothing to look for
+     # Fix for all local links to not mislead Pandoc into grabbing other files when there is nothing to look for
      # issue reported in Pandoc : https://github.com/jgm/pandoc/issues/3619
      sed -i -r 's,]:,]\\:,g' "${TEMP_FILE}"
      sed -i -r 's,\/#,,g' "${TEMP_FILE}"
 
+     # echo file -I "${TEMP_FILE}"
+
     # Set name for last folder
     if [ -z "${PDF_FILE_NAME}" ]
     then
       PDF_DEST_DIR=''
       PDF_FILE_NAME="MesosphereDCOS"
     fi
 
-    #  scripts/pandocpdf.sh "${TEMP_FILE}" "${PDF_DEST_DIR}"/"${PDF_FILE_NAME}" "${INPUT_FOLDER}"
-     # Pandoc gets the string of files and outputs the pdf.
-     echo "scripts/pandocpdf.sh ${TEMP_FILE} ${PDF_DEST_DIR}/${PDF_FILE_NAME}" >> "${PARALLEL_TEMPFILE}"
+    scripts/pandocpdf.sh "${TEMP_FILE}" "${PDF_DEST_DIR}"/"${PDF_FILE_NAME}" "${INPUT_FOLDER}"
+    # Pandoc gets the string of files and outputs the pdf.
+    # echo "scripts/pandocpdf.sh ${TEMP_FILE} ${PDF_DEST_DIR}/${PDF_FILE_NAME}" >> "${PARALLEL_TEMPFILE}"
```

Comment:
but without parallel i probably gets slower again. Is there a reason to drop it?